src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,148 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> head;
+    private Node<T> tail;
+
     @Override
     public void add(T value) {
+        Node<T> node = tail;
+        Node<T> newNode = new Node<>(node, value, null);
+        tail = newNode;
+        if (node == null) {
+            head = newNode;
```

Comment:
you need to check if head of your linked list is null, and if yes, assign it to newNode, as well as tail